src/harness/reference_models/ppa/ppa_test.py

```diff
@@ -89,18 +89,18 @@ def test_ClippedPpaByCensus(self):
     self.assertAlmostSamePolygon(
         utils.ToShapely(ppa_zone), expected_ppa, 0.001)
 
-  def test_ClippedPpaByCensusWithSmallHoles(self):
+  def test_ClippedPpaByCountyWithSmallHoles(self):
     # Configuring for -96dBm circle above 40km
     wf_hybrid.CalcHybridPropagationLoss = testutils.FakePropagationPredictor(
         dist_type='REAL', factor=1.0, offset=(96+30-0.1) - 45.0)
 
     ppa_zone = ppa.PpaCreationModel(TestPpa.devices[1:], TestPpa.pal_records[1:])
     ppa_zone = json.loads(ppa_zone)
 
-    census_zone = utils.ToShapely(
-        drive.census_tract_driver.GetCensusTract('06027000100')
+    county_zone = utils.ToShapely(
+        drive.county_driver.GetCounty('06027000100')
```

Comment:
Guess fipsCode here should change to first 5 digits '06027' to get corresponding county shape file.

Reply:
for this unit test, the name can be any string as long as the file exists in \src\harness\reference_models\ppa\test_data, I don't think it is necessary to have a name  with 5 digits which reflects a real name of a county as it is only a fake name for this unit test, don't you agree?

Reply:
Agree test won't fail as long as file exists in the test_data directory. Don't you think it would be good to stay consistent and use real identifiers even for unit tests? Anyways, I leave the decision to you and other approvers as you see fit.

All other changes look good.

Reply:
thank you!  the update is done.